rules/catering-to-audience/rule.md

```diff
@@ -0,0 +1,115 @@
+---
+type: rule
+title: Do you cater to your audience?
+uri: catering-to-audience
+authors:
+  - title: Piers Sinclair
+    url: https://www.ssw.com.au/people/piers-sinclair
+  - title: Ulysses Maclaren
+    url: https://www.ssw.com.au/people/uly
+created: 2022-08-24T00:53:23.043Z
+guid: 158ca58a-ee03-476b-b97a-a1bcaa57bd24
+---
+Often when you are talking with others, it is easy to forget they have a different background and experience to you. Then, once you start explaining something to them, they become easily lost. So, it is crucial to think about your audience before talking.
+
+Some examples of what people might care about include:
+
+<!--endintro-->
+
+* Product Owner - May not care about all the technical details, but cares a lot about PBI progress, roadblocks etc

```

Comment:
Should this be reordered like the others to what they care about first?
"Cares a lot about PBI progress, roadblocks and business value, but may not be as concerned with the technical details"